Prefabs/Tanks/Player2Tank.tscn

```diff
@@ -0,0 +1,20 @@
+[gd_scene load_steps=5 format=2]
+
+[ext_resource path="res://Prefabs/Tanks/Tank.tscn" type="PackedScene" id=1]
+[ext_resource path="res://Assets/Images/Tank.png" type="Texture" id=2]
+[ext_resource path="res://Scripts/Player2Tank.gd" type="Script" id=3]
+
+[sub_resource type="RectangleShape2D" id=1]
+extents = Vector2( 11.5, 16 )
+
+[node name="Player2Tank" instance=ExtResource( 1 )]
+script = ExtResource( 3 )
+
+[node name="Sprite" parent="." index="0"]
+modulate = Color( 0, 0.984314, 0.360784, 1 )
```

Comment:
For now, I just changed the tint, I wanted to this first and refactoring the sprite after this :)